homeassistant/components/bond/const.py

```diff
@@ -10,3 +10,10 @@
 HUB = "hub"
 BPUP_SUBS = "bpup_subs"
 BPUP_STOP = "bpup_stop"
+
+SERVICE_SET_FAN_SPEED_BELIEF = "set_fan_speed_belief"
+SERVICE_SET_POWER_STATE_BELIEF = "set_switch_power_state_belief"
+SERVICE_SET_LIGHT_POWER_STATE_BELIEF = "set_light_power_state_belief"
+SERVICE_SET_LIGHT_BRIGHTNESS_STATE_BELIEF = "set_light_brightness_state_belief"
```

Comment:
Can make this more consistent? Maybe `set_fan_speed_state_belief` or maybe drop `_state` since I think that is already implied and would make it less to type out.

Reply:
I'll drop state to make them shorter